src/desktop/main.js

```diff
@@ -73,22 +71,22 @@ let windowState = {
 };
 
 try {
-    const data = electronSettings.get('reduxPersist:settings');
     const windowStateData = electronSettings.get('window-state');
     if (windowStateData) {
         windowState = windowStateData;
     }
-    settings = JSON.parse(data) || {};
 } catch (error) {}
 
 /**
  * Temporarily disable proxy if not overridden by settings
  */
-
-if (settings.ignoreProxy) {
-    app.commandLine.appendSwitch('auto-detect', 'false');
-    app.commandLine.appendSwitch('no-proxy-server');
-}
+try {
+    const ignoreProxy = electronSettings.get('ignore-proxy');
+    if (ignoreProxy) {
+        app.commandLine.appendSwitch('auto-detect', 'false');
+        app.commandLine.appendSwitch('no-proxy-server');
+    }
+} catch (error) {}
```

Comment:
Do we need to do anything on catch?

Reply:
Can't think of anything we could do. This can fail if the settings cannot be read from the disk, nothing much we can do in that case.